jax/_src/numpy/polynomials/polyutils.py

```diff
@@ -0,0 +1,81 @@
+import operator
+import functools
+import warnings
+from jax._src.numpy.lax_numpy import asarray
+from functools import partial
+from jax import jit
+from jax._src.numpy.util import _check_arraylike, _promote_dtypes, _promote_dtypes_inexact
+
+import jax.numpy as jnp
+import jax.lax as lax
+
+__all__ = [
+    'as_series', 'trimseq',
+    'trimcoef', 'getdomain', 'mapdomain', 'mapparms']
+
+#Not compatible with JIT
+def trimseq(seq):
+    if len(seq) == 0:
+        return seq
+    else:
+        for i in range(len(seq)-1, -1, -1):
+            if seq[i] != 0:
+                break
+        return seq[:i+1]
+
+#Works with JIT if static_argnames('trim',) and trim=False
+@partial(jit, static_argnames=('trim',))
+def as_series(alist, trim=False):
+    arrays = [jnp.array(a, ndmin=1, copy=False) for a in alist]
+    if min([a.size for a in arrays]) == 0:
+        raise ValueError("Coefficient array is empty")
+    if any(a.ndim != 1 for a in arrays):
+        raise ValueError("Coefficient array is not 1-d")
+    if trim:
+        arrays = [trimseq(a) for a in arrays]
```

Comment:
This line will fail because of the `jit` decoration above and the incompatbility of `trimseq` with JIT